tests/ui/conftest.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import os
+from functools import partial, partialmethod
+import pytest
+
+
+logging.TRACE = 5
+logging.addLevelName(logging.TRACE, "TRACE")
+logging.Logger.trace = partialmethod(logging.Logger.log, logging.TRACE)
+logging.trace = partial(logging.log, logging.TRACE)
+
+
+@pytest.fixture(scope="session")
+def logger() -> logging.Logger:
+    return logging.getLogger("ui-tests")
+
+
+@pytest.fixture(scope="session")
```

Comment:
already implemented https://playwright.dev/python/docs/test-runners#configure-base-url

Reply:
so should I remove this block and add --base-url flag to the make command??

Reply:
yes, unless we have a serious reason for reimplementing existing functionality